data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+			`	You look at the figure out of the corner of your eye and connect the whispered voice to the squirrelly muzzle poking out. To your surprise it is Sayari.`
+			`	"I should not be coming to you with this, so you must promise to say nothing of this to anyone just yet."`
+			choice
+				`	"Of course."`
+			`	"A very sweet friend of mine, who is much too enthusiastic and perhaps led by the vagaries of youth but nonetheless quite astute, keeps harping on to me about this theory of hers that a smuggling ring is kidnapping Hai and transporting them into human space."`
+			`	She pauses a moment for you to absorb this, "Of course, a thorough investigation has occurred on our side and no evidence of such things has turned up, but..."`
+			`	"Well, you see we cannot investigate your space openly or easily. I have paid it little mind, written it off as one of her eccentricities, but things are very tense right now. The Council of Elders wants to maintain our secrecy, the human diplomats are willing to accomodate us, but we sense their governments grow tired of maintaining the fiction. We are well aware that human intelligence services are highly competent, and motivated by many different factors. It worries me that one of your factions may not be wholly honest, and may have knowledge they intend to use as ammunition if things don't work out well for them."`
+			`	You have not had much exposure to this elected "Council of Elders", but it seems they must hold some kind of final authority over decisions.`
+			choice
+				`	"But what may I do about it?"`
+			`	"You are well-known, and must be canny to have got where you are. A Hai in trouble might recognise you, or you might perceive them when others would not. It would be in the interests of both our people to be sure that there is no such issue. Would you look for me? For both our sakes?"`
+			branch "about that"
+				"hai slave prereq" >= 1
+			choice
+				`	"Sure, I can have a look around."`
+					accept	
+			label "about that"
+			`	"Well, about that," you say, "I think I might have already come across some evidence of this before."`
+			`	Sayari's hood nearly slips off as she looks up at you with a horrified expression, "Surely not?!"`
+			`	You rush to reassure her, "Nothing concrete, of course I shall go and have a look to be sure."`
+				accept
+	to complete
+		"hai slave prereq" >= 3
+	on complete
+		log `You found proof of more kidnapped Hai. Sayari's fears appear to have been entirely justified...`
+
+mission "Hai Leaks Response 7"
+	landing
+	invisible
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent" "Hai"
+		not attributes "uninhabited"
+	to offer
+		has "Hai Leaks Response 6: active"
+		"hai slave prereq" >= 3
+	on offer
+		conversation
+			`You think you have enough evidence of kidnapped Hai to be sure of the bad news you bring. There may be other Hai you could still save, but you have enough information for Sayari.`
```

Comment:
```suggestion
			`You think you have enough evidence of kidnapped Hai to be sure that something's going on here. There may be other Hai you could still save, but you should go tell Sayari first to see how to proceed.`
```

Reply:
I think it's pretty clear that it's bad news though. 
It was "Can you find evidence of this?"
"Yes."
"Well crap."

Reply:
Suggestion updated.